pkg/chartverifier/reportBuilder.go

```diff
@@ -102,6 +111,9 @@ func (r *reportBuilder) Build() (*Report, error) {
 			}
 		}
 	}
+
+	r.Report.Metadata.ToolMetadata.Digests.Package = GetPackageDigest(r.Report.Metadata.ToolMetadata.ChartUri)
```

Comment:
Is this going to show digest: "" then if there was an issue? Should we check first if GetPackageDigest return non empty?

Reply:
No, tested that "" counts as empty for omitempty.

Reply:
OK perfect thanks